src/components/image/index.js

```diff
@@ -1,23 +1,44 @@
 import React from "react"
 import PropTypes from "prop-types"
 import Img from "gatsby-image"
-import classNames from "classnames/bind"
+import classNames from "classnames"
 import dig from "object-dig"
+
 import styles from "./styles.module.scss"
 
 const Image = ({ alt, className, src, ...props }) => {
-  const classes = classNames(styles.cl_image, { [className]: className })
+  const defaultAltAttribute = image =>
+    image
+      .split("/")
+      .pop()
+      .split(".")
+      .slice(0, -1)
+      .join(".")
```

Comment:
We should be testing the logic of this.

Reply:
What I'd like to see here is that we pull the logic out into an exported function that we can then import into the spec file and test the logic of it.

Happy to pair if that doesn't make sense.